src/Store.js

```diff
@@ -27,6 +27,7 @@ export default class Store {
     Model = null;
     api = null;
     __repository;
```

Comment:
Is the `__repository` property still used on the store after your fix? If not, please delete.